Demo/Demo/Models/ClientIDRequest.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 import CorePayments
 
-struct ClientIDRequest: APIRequest {
+struct ClientIDRequest {
 
```

Comment:
Sammy, on my draft PR for vault without purchase, I had reused this createUrlRequest for setup token call with APIRequest protocol. I have to have some other protocol so we can take in different request structs to create url for demo app.

Reply:
I addressed it in this commit. It's redundant but I will consolidate if we have another protocol.

Reply:
The Demo should never have been using `APIRequest`